src/internal/components/dropdown-status/interfaces.ts

```diff
@@ -25,6 +25,11 @@ export interface DropdownStatusProps {
    * Use the `onLoadItems` event to perform a recovery action (for example, retrying the request).
    **/
   recoveryText?: string;
+
+  /**
+   * Provides a text alternative for the error icon in the error alert.
```

Comment:
Documentation consistency: Currently we use
- "Provides a text alternative for the error icon in the **error alert**." in the Form component ([link to source](https://github.com/cloudscape-design/components/blob/53d8ce9339ec5fb04d341df17ba238aa4c3bfb34/src/form/interfaces.ts#L23))
- "Provides a text alternative for the error icon in the **error message**." in the Form Field component ([link to source](https://github.com/cloudscape-design/components/blob/72dfb082b6ea11dd4f48bc4b62a42c8f990fb42d/src/form-field/interfaces.ts#L78))

I think for the Form component it makes sense because it's for the icon shown in the alert ([link to playground example](https://cloudscape.design/components/form/?tabId=playground&example=with-errors)).

Errors in the Multiselect, Select and Autosuggest are not sown in the alert component, so I suggest to update it to "... error icon in the **error message**."